cmplibrary/src/main/res/raw/js_receiver.js

```diff
@@ -2,18 +2,25 @@ function isFromPM(payload) {
     return payload.fromPM || (payload.settings && payload.settings.vendorList != null);
 }
 
-function getPmIdFromURL(url) {
-    return url ? url.match(/[?&]message_id(=([^&#]*)|&|#|$)/)[2] : null;
+function getPmIdAndPmTabFromURL(url) {
+    let pmId = url ? url.match(/[?&]message_id(=([^&#]*)|&|#|$)/)[2] : null;
+    let pmTabShow = null;
+    if(url &&  url.match(/[?&]pmTab(=([^&#]*)|&|#|$)/) && url.match(/[?&]pmTab(=([^&#]*)|&|#|$)/)[2]){
+       pmTabShow = url.match(/[?&]pmTab(=([^&#]*)|&|#|$)/)[2];
+     }
+    return [pmId, pmTabShow];
```

Comment:
here you can return a object:
```JavaScript
{pmId,pmTabShow}
```
so later get the values by the keys instead :)

Reply:
Good suggestion. Done.